x-pack/legacy/plugins/security/public/views/management/role_mappings/model/except_all_rule.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { RuleGroup } from './rule_group';
+import { Rule } from './rule';
+
+/**
+ * Represents a group of rules in which at least one must evaluate to false.
+ */
+export class ExceptAllRule extends RuleGroup {
+  constructor(private rules: Rule[] = []) {
+    super();
+  }
+
+  /** {@see RuleGroup.getRules} */
+  public getRules() {
+    return [...this.rules];
+  }
+
+  /** {@see RuleGroup.getDisplayTitle} */
+  public getDisplayTitle() {
+    return i18n.translate('xpack.security.management.editRoleMapping.exceptAllRule.displayTitle', {
+      defaultMessage: 'Any are false',
```

Comment:
It seems like this should be swapped with the message in `except_any_rule`?

Reply:
hmm, my understanding is the [`except` clause](https://www.elastic.co/guide/en/elasticsearch/reference/current/role-mapping-resources.html#role-mapping-resources) negates the rule that follows. The `all` rule states that _all of the following rules must be true_. Negating that would be _all of the following rules must not be true_, which is how I arrived at `Any are false` for this message.

It's very possible that I'm missing something here though 😄

Reply:
Oh, duh, yep.